### PR TITLE
FAQ update: force USB2 mode

### PR DESCRIPTION
**Pull request by cafemoloko**
update on how to force USB2 mode using code (rather than CLI)

### CODE REVIEW

**Review by VanDavv — APPROVED** [review]

**Review by Erol444 — APPROVED** [review]

**Comment by Luxonis-Brandon** [comment]
So I'm thinking we should leave in the CLI option as well with depthai_demo.py.  I think that still works in Gen2 depthai_demo.py right @VanDavv ?

**Comment by VanDavv** [comment]
Surprisingly there is no fusb2 flag in the demo script, I'll add it quickly in a minute and circle back with PR (and check which other flags might be missing)

Regarding the PR itself, the changes from @cafemoloko just add additional info about USB mode, they do not remove the mention about CLI flags

File: source/pages/faq.rst [hunk 1]
```diff
@@ -1148,13 +1148,20 @@ How Do I Force USB2 Mode?
 
 USB2 Communication may be desirable if you'd like to use extra-long USB cables and don't need USB3 speeds.
 
-To force USB2 mode, simply use the `-fusb2` (or `--force_usb2`) command line option as below:
+You can force USB2 mode by setting :code:`usb2mode` to :code:`True` when creating the device (note - it works for gen2):
+
+.. code-block:: python
+
+  dai.Device(pipeline, usb2mode=True)
```
Comment (VanDavv) [inline source/pages/faq.rst:1155]: typo, rename to `usb2Mode`

**Review by VanDavv — CHANGES_REQUESTED** [review]

**Comment by Luxonis-Brandon** [comment]
Ah cool thanks.  I mis-read the diff file I think.

File: source/pages/faq.rst (hunk 1 above)
Comment (cafemoloko) [reply source/pages/faq.rst]: @VanDavv typo fixed, thanks

File: source/pages/faq.rst [hunk 2]
```diff
@@ -1148,13 +1148,20 @@ How Do I Force USB2 Mode?
 
 USB2 Communication may be desirable if you'd like to use extra-long USB cables and don't need USB3 speeds.
 
-To force USB2 mode, simply use the `-fusb2` (or `--force_usb2`) command line option as below:
+You can force USB2 mode by setting :code:`usb2mode` to :code:`True` when creating the device (note - it works for gen2):
```
Comment (VanDavv) [inline source/pages/faq.rst:1151]: Typo

**Review by VanDavv — CHANGES_REQUESTED** [review]

**Review by VanDavv — APPROVED** [review]

**Pull request merged**